client/src/components/Details/Reviewpage.js

```diff
@@ -9,6 +9,10 @@ const FilmDetail = ({ filmId }) => {
 
   useEffect(() => {
     const fetchReviews = async () => {
+
+      console.log("Testing URL :", `https://www.googleapis.com/youtube/v3/search?part=snippet&maxResults=10&q=${MovieDetail ? MovieDetail.original_title: ""} review&type=video&key=AIzaSyDcPfOvNAZ2PwHX2lrX1oHfI_D_TJLkNgk`)
+
+
       const response = await axios.get(
         `https://www.googleapis.com/youtube/v3/search?part=snippet&maxResults=10&q=${MovieDetail ? MovieDetail.original_title: ""} review&type=video&key=AIzaSyDcPfOvNAZ2PwHX2lrX1oHfI_D_TJLkNgk`
```

Comment:
Use environment variables in for sensitive details, (https://trekinbami.medium.com/using-environment-variables-in-react-6b0a99d83cf5) read the attached blog and implement the same.